marconi-chain-index/src/Marconi/ChainIndex/Experimental/Run.hs

```diff
@@ -25,25 +30,43 @@ run appName = do
       includeScript = True
       socketPath = Cli.optionsSocketPath $ Cli.commonOptions o
       networkId = Cli.optionsNetworkId $ Cli.commonOptions o
+      retryConfig = Cli.optionsRetryConfig $ Cli.commonOptions o
```

Comment:
Is it useful to have it in this PR?

Reply:
No, but it was pretty trivial to add it as it was a copy-paste from the old indexer. However, it makes more sense to put in a separate PR.